site/zenodo_rdm/legacy/deserializers/schemas.py

```diff
@@ -9,6 +9,7 @@
 
 from datetime import date
 
+from invenio_records.dictutils import clear_none
```

Comment:
should open an issue, we have plenty of `drop_nones` usage. Very similar if not identical in functionality:
- https://github.com/inveniosoftware/invenio-rdm-migrator/blob/bd88485a7cb3d09a69d2b802e0029082dc232188/invenio_rdm_migrator/transform/base.py#LL36C1-L36C1
- https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/dictutils.py#L14

Reply:
I saw that one in invenio-rdm-migrator but I didn't change it because I assumed that we want ideally not to depend on `invenio-records`....

Reply:
Yes, dependency is kind of an issue, I dont know what we should do... in the end this module is `invenio-` so it should be ok to depend. On the other hand it would make tests slower if we bring in app fixtures and co. Im conflicted. To be evaluated.